src/main/daml/Daml/Finance/Interface/Types/Date/RollConvention.daml

```diff
@@ -24,10 +24,20 @@ data PeriodEnum
   -- ^ Year
   deriving (Eq, Show)
 
+-- | An enum type to either specify a period or a single term.
+data ScheduleFrequencyEnum
+  = RegularPeriod PeriodEnum
```

Comment:
```suggestion
  = Tenor PeriodEnum
```

What about `Tenor`? It seems more descriptive than `RegularPeriod`

Reply:
Also, I think that the right type to use here would be `PeriodEnum * Int` as a `Period` is defined by the unit and the multiplier